include/mapbox/io/io.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <fstream>
+#include <sstream>
+#include <string>
+
+#include "mapbox/util/expected.hpp"
```

Comment:
```suggestion
#include <mapbox/util/expected.hpp>
```

Reply:
This method is normally used to include library header files.

Reply:
This is like a flag:
`""` used for includes in the same library
`<>` used for includes from other libraries

Reply:
According to google standard that we use https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Quotation marks are used to tell the preprocessor to search first in the same directory as the file containing the directive, then proceed to search on the include paths. This is not needed and is confusing to read IMO.

Reply:
Let's make codestyle description then. In clang-format i can figure out that we use google codestyle. 
My opinion that if we separate includes by two types it much easier to understand from which library that file was included

Reply:
Fair point - let's introduce `clang-format` once we get all the individual libraries onboard.